minter-suite/seadrop-integration.md

```diff
@@ -0,0 +1,92 @@
+---
+order: -20
+---
+
+# SeaDrop Integration
+
+This page provides an overview of an integration solution for the Art Blocks Shared Minter Suite with SeaDrop. This is a special integration that allows Art Blocks projects to be minted and sold on OpenSea via SeaDrop, and is not a standard integration with the Shared Minter Suite.
```

Comment:
change `with OpenSea` to `with OpenSea's SeaDrop <contract/infra/system>`

Reply:
updated in [142c186](https://github.com/ArtBlocks/artblocks-docs/pull/314/commits/142c1866a6bf1f6cad6189f26aef34be0023f256)